lua/silicon.lua

```diff
@@ -36,21 +36,14 @@ M.start = function(args)
 	for k, v in pairs(M.opts) do
 		if k == "command" or k == "gobble" then
 			-- no-op
-		elseif k == "language" then
-			table.insert(cmdline, '--language')
+		elseif k == "language" or k == "output" or k == "window_title" then
+			table.insert(cmdline, "--" .. string.gsub(k, "_", "-"))
 			if type(v) == "function" then
-				table.insert(cmdline, v())
+				value = v()
 			else
-				table.insert(cmdline, v)
-			end
-		elseif k == "output" then
-			table.insert(cmdline, '--output')
-			if type(v) == "function" then
-				filename = v()
-			else
-				filename = v
+				value = v
 			end
-			table.insert(cmdline, filename)
+			table.insert(cmdline, value)
```

Comment:
Then here:
```
if k == "output" then filename = value end
```

Reply:
That's my bad, didn't realize it was being used elsewhere. Fixed